code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -19,6 +19,7 @@
 	icon_state = "dispenser"
 	use_power = 0
 	idle_power_usage = 40
+	layer = BELOW_OBJ_LAYER
```

Comment:
does this make them still render above tables?

Reply:
BTW, how about set that layer for all machinery?